MAUI/Calendar/overview.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: post
+title: About .NET MAUI Calendar control | Syncfusion
+description: Learn here the overview of Syncfusion .NET MAUI Calendar(SfCalendar) control, its basic features and functionalities of calendar.
+platform: maui
+control: SfCalendar
+documentation: ug
+---
+
+# Overview of .NET MAUI Calendar (SfCalendar)
+
+The Syncfusion .NET MAUI Calendar (SfCalendar) allows user to select a Single date, Multiple dates or a Range of dates. It provides Month, Year, Decade, and Century view options for quickly navigate to the desired date. It supports minimum dates, maximum dates, enable past dates, and disabled dates to restrict the date selection.
```

Comment:
Syncfusion .NET MAUI Calendar (SfCalendar) is a calendar control that allows users to select a Single date, Multiple dates, or a Range of dates. It provides various view options such as a Month, Year, Decade, and Century, which allow users to quickly navigate to the desired date. It also supports minimum dates, maximum dates, enable past dates, and disabled dates to restrict the date selection.

Reply:
Changed the overview content